submissions/KostiantynOkhorzin/html-css-popup/index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Popup</title>
+    <link rel="shortcut icon" href="icons/favicon.png" type="image/x-icon">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+    <body>
+        <header class="header">
+            <nav class="navbar">
+                <ul class="navbar__menu">
+                    <li class="navbar__item">
+                        <a href="#" class="navbar__link">Mail</a>
+                    </li>
+                    <li class="navbar__item">
+                        <a href="#" class="navbar__link">Images</a>
+                    </li>
+                    <li class="navbar__item popap">
+                        <input class="popap__input" id="popap__trigger" type="checkbox">
+                        <label class="popap__trigger" for="popap__trigger">
+                            <img class="popap__trigger-icon" src="icons/popup-button.png" alt="icon">
+                        </label>
+                        <div class="popap__body">
+                            <input class="popap__input-more" id="popap__more" type="checkbox">
```

Comment:
You need move this `input` because when you navigate with `tab`, focus should be at icons first, and on `More` after icons.